src/components/main/index.tsx

```diff
@@ -160,7 +160,11 @@ const Main: React.FC = (): JSX.Element => {
               text="mph"
               additionalInfo={
                 <WindDirection>
-                  <WindStatusBadge src={WindStatusIcon} alt="wind-icon" />
+                  <WindStatusBadge
+                    rotate={`${setWindDirection(wind_direction_compass)}`}
```

Comment:
you don't need util for calculation of rotation, use wind_direction property from consolidated_weather